nestkernel/per_thread_bool_indicator.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ *  per_thread_bool_indicator.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "per_thread_bool_indicator.h"
+
+namespace nest
+{
+
+BoolIndicatorUInt64::BoolIndicatorUInt64()
+  : status_( false_uint64 )
+{
+}
+
+BoolIndicatorUInt64::BoolIndicatorUInt64( const bool status )
+  : status_( status )
+{
+}
+
+bool
```

Comment:
Move to header file and `inline`, the same for `is_false()` and the `set_*()` methods.

Reply:
Done.